.github/dependabot.yml

```diff
@@ -0,0 +1,38 @@
+---
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: daily
+    allow:
+      - dependency-name: mongodb
+      - dependency-name: bson
+      - dependency-name: "@mongosh/*"
+      - dependency-name: "@mongodb-js/*"
+      - dependency-name: mongodb-*
+      - dependency-name: bson-*
+    groups:
+      driver:
+        patterns:
+          - mongodb
+          - bson
+      compass:
+        patterns:
+          - "@mongodb-js/compass-*"
+          - mongodb-data-service
+          - bson-transpilers
+          - "@mongodb-js/connection-form"
+      devtools-shared:
+        patterns:
```

Comment:
these 3 are independent enough that there is probably no need to have them all grouped